app.R

```diff
@@ -1,49 +1,103 @@
+  group_by(sp) %>%
+  summarize(avg_dbh_sp = mean(dbh),
+            census_year = mean(CensusID))
+
+stem_joined <- rbind(stem1, stem2, stem3)
+
+genus_dbh <- stem_joined %>%
+  inner_join(scbi_spptable, by = "sp") %>%
+  select(Genus, avg_dbh_sp, census_year) %>%
+  group_by(Genus, census_year) %>%
+  summarize(avg_dbh = mean(avg_dbh_sp))
+
+year.labs <- c("2008", "2013", "2018")
+names(year.labs) <- c("1", "2", "3")
+
+
 
-# Define UI for application that draws a histogram
 ui <- fluidPage(
 
-    # Application title
-    titlePanel("Old Faithful Geyser Data"),
 
-    # Sidebar with a slider input for number of bins
+    titlePanel("Average DBH for Each Genus in the SCBI Census"),
+
+
     sidebarLayout(
-        sidebarPanel(
-            sliderInput("bins",
-                        "Number of bins:",
-                        min = 1,
-                        max = 50,
-                        value = 30)
-        ),
-
-        # Show a plot of the generated distribution
-        mainPanel(
-           plotOutput("distPlot")
-        )
+      sidebarPanel(
+        selectInput("census_year", "Census Year",
+                    choices = c("2008" = "1","2013" = "2", "2018" = "3"),
+                    selected = "3"),
+        checkboxGroupInput("Genus", "Genus",
+                           choices = c("Acer","Ailanthus","Amelanchier","Asimina",
+                                       "Berberis","Carpinus","Carya","Castanea","Celtis",
+                                       "Cercis","Chionanthus","Cornus","Corylus",
+                                       "Crataegus","Diospyros","Elaeagnus","Euonymus",
+                                       "Fagus","Fraxinus","Hamamelis","Ilex","Juglans",
+                                       "Juniperus","Lindera","Liriodendron","Lonicera","Nyssa",
+                                       "Paulownia","Pinus","Platanus","Prunus","Quercus",
+                                       "Rhododendron","Robinia","Rosa","Rubus","Sambucus",
+                                       "Sassafras","Tilia","Ulmus","Viburnum"),
+                           selected = c("Acer","Ailanthus","Amelanchier","Asimina",
+                                        "Berberis","Carpinus","Carya","Castanea","Celtis",
+                                        "Cercis","Chionanthus","Cornus","Corylus",
+                                        "Crataegus","Diospyros","Elaeagnus","Euonymus",
+                                        "Fagus","Fraxinus","Hamamelis","Ilex","Juglans",
+                                        "Juniperus","Lindera","Liriodendron","Lonicera","Nyssa",
+                                        "Paulownia","Pinus","Platanus","Prunus","Quercus",
+                                        "Rhododendron","Robinia","Rosa","Rubus","Sambucus",
+                                        "Sassafras","Tilia","Ulmus","Viburnum"),
+                           inline = FALSE),
+      ),
+      plotOutput("avg_dbh")
```

Comment:
You should revert this line to what it was originally. See how on line 90 below the plot gets saved to `output$distplot` and not `output$avg_dbh`